strings.json

```diff
@@ -2825,9 +2838,11 @@
   },
   "portfolio_splitsHelp": {
     "en": "Examples:\\n\\nA 4-for-1 split will multiply shares owned by 4 and divide cost paid per share by 4.\\n\\nA 1-for-4 reverse split will divide shares owned by 4 and multiple cost per share by 4."
+    "tr": "Örnekler:\\n\\n%400 bölünme, sahip olunan hisseleri 4 ile çarpacak ve hisse başına ödenen maliyeti 4'e bölecektir.\\n\\n%400 ters bölünme, sahip olunan hisseleri 4'e bölecek ve hisse başına maliyeti 4'e katlayacaktır.",
```

Comment:
@ylmzer - In the sentence "%400 bölünme, sahip olunan hisseleri 4 ile çarpacak ve hisse başına ödenen maliyeti 4'e bölecektir.\\n\\n%400 ters bölünme, sahip olunan hisseleri 4'e bölecek ve hisse başına maliyeti 4'e katlayacaktır.", just wanted to double check, is "%400" the correct translation?

Reply:
Hello, In our country, splits are usually used as "%". Example: ALFAS - 700%, ASELS - 100%.
I made the translation according to this usage. I can change it if it's a problem.

Reply:
Ok thanks for the explanation! Let's leave it like this.